docs2/site/docs/migrations/migration5.md

```diff
@@ -21,3 +25,14 @@ renamed to have `Async` suffix.
 
 1. Use async methods to get or set a cache.
 2. Cache items cannot be removed anymore.
+
+### `IResolveFieldContext.Extensions` property renamed to `OutputExtensions`
+
+To clarify and differ output extensions from input extensions, `IResolveFieldContext.Extensions`
+has now been renamed to `OutputExtensions`. There has not been any change to the `GetExtension` and
```

Comment:
> There has not been any change to the `GetExtension` and
`SetExtension` thread-safe extension methods.

They will also be renamed.

Reply:
Renamed as suggested.